docs/state.md

```diff
@@ -36,8 +36,11 @@ AppStore
       updatedStamp: number // timestamp for when the data was last updated
     },
     preferences: {
+      backupNotifyCount: number, // number of times user has been reminded to backup wallet
+      backupNotifyTimestamp: number, // number of milliseconds from the last reminder until the next
+      backupSucceeded: (boolean|undefined), // was last backup successful?
       recoverySucceeded: (boolean|undefined),
-      updatedStamp: number
+      updatedStamp: number      
```

Comment:
trailing space

Reply:
Did run lint pick that up for you or did you just happen to see it